src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -229,24 +228,29 @@ def _flexible_server_params(command_group):
         with self.argument_context('{} flexible-server create'.format(command_group)) as c:
             # Add create mode as a parameter
             if command_group == 'postgres':
-                c.argument('tier', default='GeneralPurpose', options_list=['--tier'], validator=tier_validator,
+                c.argument('tier', default='GeneralPurpose', options_list=['--tier'],
                            help='Compute tier of the server. Accepted values: Burstable, GeneralPurpose, Memory Optimized ')
-                c.argument('sku_name', default='Standard_D2s_v3', options_list=['--sku-name'], validator=pg_sku_name_validator,
+                c.argument('sku_name', default='Standard_D2s_v3', options_list=['--sku-name'],
                            help='The name of the compute SKU. Follows the convention Standard_{VM name}. Examples: Standard_D4s_v3 ')
-                c.argument('storage_mb', default='128', options_list=['--storage-size'], type=int, validator=pg_storage_validator,
+                c.argument('storage_mb', default='128', options_list=['--storage-size'], type=int,
                            help='The storage capacity of the server. Minimum is 32 GiB and max is 16 TiB.')
-                c.argument('version', default='12', options_list=['--version'], validator=pg_version_validator,
+                c.argument('backup_retention', default=7, type=int, options_list=['--backup-retention'],
+                           help='The number of days a backup is retained. Range of 7 to 35 days. Default is 7 days.',
+                           validator=retention_validator)
+                c.argument('version', default='12', options_list=['--version'],
```

Comment:
quick question: how can user know which kind of version is accepted？

Reply:
> quick question: how can user know which kind of version is accepted？

Do you mean how they can find out which are the valid versions ?

Reply:
If they pass in a wrong value, they will get a Validation error with the set of acceptable values.